prepare_vscode.sh

```diff
@@ -85,13 +84,15 @@ fi
 
 setpath() {
   { set +x; } 2>/dev/null
-  echo "$( cat "${1}.json" | jq --arg 'path' "${2}" --arg 'value' "${3}" 'setpath([$path]; $value)' )" > "${1}.json"
+  jsonTmp=$(jq --arg 'path' "${2}" --arg 'value' "${3}" 'setpath([$path]; $value)' "${1}.json")
+  echo "$jsonTmp" > "${1}.json" && unset jsonTmp
```

Comment:
`"${jsonTmp}"` instead of `"$jsonTmp"` (for consistency)

Reply:
I will change this, however I will note that this can lead to confusion. When I read `${}` I assume an expansion is intended unless it's obvious it's intended as a deliminator. i.e `"${HOME}/git"`.